app/src/main/java/com/infomaniak/drive/data/api/ApiRoutes.kt

```diff
@@ -277,12 +280,36 @@ object ApiRoutes {
 
     fun startUploadSession(driveId: Int) = "${uploadSessionUrl(driveId)}/start"
 
-    fun addChunkToSession(uploadHost: String, driveId: Int, uploadToken: String) =
+    private fun addChunkToSession(uploadHost: String, driveId: Int, uploadToken: String) =
         "$uploadHost/3/drive/$driveId/upload/session/$uploadToken/chunk"
 
     fun closeSession(driveId: Int, uploadToken: String) = "${uploadSessionUrl(driveId)}/$uploadToken/finish"
 
-    fun uploadFile(driveId: Int) = "${driveURLV2(driveId)}/upload"
+    private fun uploadFile(driveId: Int) = "${driveURL(driveId)}/upload"
```

Comment:
Maybe it would be better to specify here that you return a URL ? Like `uploadChunkUrl` ? I noticed that we have both naming convention.